editor/gui/editor_file_dialog.cpp

```diff
@@ -264,17 +264,19 @@ void EditorFileDialog::update_dir() {
 	}
 	dir->set_text(dir_access->get_current_dir(false));
 
+	file->set_text("");
```

Comment:
This line breaks things and clears the file each time, meaning file paths provided by the user do not work

Reply:
This will also clear the file if you change directory, which I don't think is desired

Reply:
> This will also clear the file if you change directory

It was like that before this change.

Reply:
How? If you remove that line it doesn't do that

Reply:
Hmm, I remember testing this specifically.

Reply:
Reverted this change and no, it did *not* work that way before

Reply:
@AThousandShips From what I remember this line was for clearing the file name of a file selected in one directory when moving to another one.

Reply:
Intended to work with opening files?

Reply:
Pushed an alternative fix that should keep this behavior, please test the PR #81226 which fixes the broken behavior

Reply:
Yes, one can select a directory, a .zip file or a project.godot file for the import to happen.

Reply:
That part makes sense, was just too general in this PR then :)